jni_gen/third_party/ApiSummarizer/src/main/java/com/github/hegde/mahesh/apisummarizer/elements/TypeDecl.java

```diff
@@ -0,0 +1,12 @@
+package com.github.hegde.mahesh.apisummarizer.elements;
+
+public interface TypeDecl {
+  // can be EnumType, ClassType, InterfaceType, or ArrayType
```

Comment:
Punctuation. Start with a capital, and with a dot.

(Or remove the comment, the DeclKind is an enum, so it's self documenting what kinds it can be.)